@xen-orchestra/web/src/remote-resources/use-xo-alarm-collection.ts

```diff
@@ -1,10 +1,14 @@
 import { useXoCollectionState } from '@/composables/xo-collection-state/use-xo-collection-state.ts'
+import { watchCollectionWrapper } from '@/utils/sse.util'
 import { defineRemoteResource } from '@core/packages/remote-resource/define-remote-resource.ts'
 import type { XoAlarm } from '@vates/types'
 import { useSorted } from '@vueuse/core'
 
+const alarmFields: (keyof XoAlarm)[] = ['id', 'time', 'body', 'object', 'type'] as const
+
 export const useXoAlarmCollection = defineRemoteResource({
-  url: '/rest/v0/alarms?fields=id,time,body,object',
+  url: '/rest/v0/alarms?fields='.concat(alarmFields.toString()),
```

Comment:
I suggest to be explicit using `join()` instead of relying on implicit join.

```suggestion
  url: `/rest/v0/alarms?fields=${alarmFields.join(',')}`,
```